### DISCUSSION TITLE
Error: After trying to connect as different user.

### DISCUSSION THREAD

**Discussion by hepaestus** [AntlerVC/firetable#931] [Support / Q&A] [answered]
**Describe the bug**
After manually deleting the _rowy_ data collection, my attempt to reconnect as a different user caused a failure to create the project.
I cannot enter the project to delete it. I cannot connect to the Firebase project at all any more.
Thanks.

This is in Mozilla latest on Windows 10

**Comment by BeeBombshell** [comment, marked as answer]
Hey @hepaestus. I'm wondering why this isn't working for you. Perhaps you could email us at [support@rowy.io](mailto:support@rowy.io) so we can set up a short call to help you out.

**Reply by hepaestus** [reply]
Thanks for the reply... 
I would like to figure this out but can't right now.  I'll get back to you soon.

Thanks.
P.
